packages/react-native-editor/__device-tests__/helpers/caps.js

```diff
@@ -9,6 +9,7 @@ const ios = {
 	processArguments: {
 		args: [ 'uitesting' ],
 	},
+	autoLaunch: false,
```

Comment:
This capability prevents Appium to automatically lunch the app. Tests will now control when it's launched.